markdown/developers/testing_with_gitpod.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Testing with Gitpod
+subtitle: Testing, Code review and Website dev.
+---
+
+# Introduction
+
+Gitpod is an open-source developer platform that can be spun up quickly from a Git repository, containing all the programs needed to run your pipelines. It has many purposes, including: testing pipelines, learning Nextflow, reviewing/editing code and live views of edited markdown/HTML for webpage development. In addition, within Gitpod, any change can be pushed to branches for review. 
```

Comment:
```suggestion
Gitpod is an open-source developer platform that can be spin up a little virtual machine quickly from a Git repository, containing all the programs needed to run your pipelines. It has many purposes, including: testing pipelines, learning Nextflow, reviewing/editing code and live views of edited markdown/HTML for webpage development. In addition, within Gitpod, any change can be pushed to branches for review. 
```

Or server.. whatever you want to call it. It would be good to specify it has a bit of computing power behind it to be clear you can actually do proper testing (i.e., to differencite with the github magic `.` VScode thing)

Reply:
I will add the limits in my review later.